src/sentry/integrations/github/urls.py

```diff
@@ -7,6 +7,7 @@
     url(
         r"^webhook/$",
         GitHubIntegrationsWebhookEndpoint.as_view(),
+        name="sentry-extensions-github-webhook",
```

Comment:
Should the github integration routes use `integration` instead of `extension` to be consistent with other integrations?